api/migrations/20250226151149-addMissionSourceOldCohort.js

```diff
@@ -0,0 +1,42 @@
+const { logger } = require("../src/logger");
+
+module.exports = {
+  async up(db) {
+    const cohorts = [
+      "2019",
+      "2020",
+      "2021",
+      "2022",
+      "Juillet 2022",
+      "Juin 2022",
+      "Février 2022",
+      "Octobre 2023 - NC",
+      "Juillet 2023",
+      "Juin 2023",
+      "Avril 2023 - B",
+      "Avril 2023 - A",
+      "Février 2023 - C",
+      "Avril 2024 - A",
+      "Avril 2024 - B",
+      "Avril 2024 - C",
+      "Février 2024 - A",
+      "Février 2024 - B",
+      "Juin 2024 - 2",
+      "Juillet 2024",
+      "à venir",
+    ];
+
+    for (const cohort of cohorts) {
+      const filter = {
+        email: { $not: /reliquat/i },
+        source: { $exists: false },
+        status: { $ne: "DELETED" },
+        cohort: cohort,
```

Comment:
```suggestion
        cohort: { $in: cohorts },
```
au lieu de faire une itération sur les cohorts, enlever le for et faire une seule requete.

Reply:
tu penses que c'est préférable niveau perf ?